apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -6103,6 +6120,13 @@ static void tc_fs_driver_ramdisk_ops_p(void)
 }
 #endif
 
+#ifdef CONFIG_AUTOMOUNT_USERFS
+void set_fs_mount_devname(char **path)
```

Comment:
1. It looks "get" is right.
2. How about delivering the value with return value?
 I mean `char *get_fs_mount_devname(void)`.

```
#ifdef CONFIG_AUTOMOUNT_USERFS
static char *TMP_MOUNT_DEV_DIR;
#else
#define TMP_MOUNT_DEV_DIR "/dev/smart1"
#endif

void itc_fs_main(void)
{
#ifdef CONFIG_AUTOMOUNT_USERFS
	TMP_MOUNT_DEV_DIR = get_fs_mount_devname();
#endif
}
```